### DISCUSSION TITLE
Login Page Not Displaying in RELEASE Mode on WebAssembly

### DISCUSSION THREAD

**Discussion by Swindler95** [unoplatform/uno#17769] [Q&A] [answered]
### Current behavior

Hello,

I am using Uno Platform 5.3.17.21 and VS 17.10.3.
I created a .NET 8.0 XAML MVVM project and test CUSTOM authentication without modifying it.
The project compiles correctly in DEBUG and RELEASE WebAssembly modes.
In DEBUG mode, the application works perfectly, but in RELEASE mode, the Login page does not appear.
I have an error message in the browser console:

`fail: Uno.UI.Dispatching.NativeDispatcher[0]
      NativeDispatcher unhandled exception
System.InvalidOperationException: NoConstructorMatch, Uno.Extensions.Authentication.Custom.CustomAuthenticationProvider`

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This is generally a linker issue, make sure to take a look at our [troubleshooting documentation](https://platform.uno/docs/articles/common-issues-all-ides.html#runtime-error-no-parameterless-constructor-defined-for-xxxx).

**Reply by Swindler95** [reply]
Thank you for your response,
But I have tried everything to make it work, I even tested on other computers and got the same result.
I simply created the test project like this:
`dotnet new unoapp -o MyProject -preset "recommended" -platforms "wasm" -platforms "windows" -auth "custom" -server -wasm-multi-threading -theme "fluent" -presentation "mvvm" -pub "MyProject"`

I don't make any modifications to the code, I just compile the project in WebAssembly RELEASE, the project compiles and launches correctly, but nothing is displayed.
Thank you for providing the documentation but I absolutely cannot find the solution to this problem.
Moreover, I tested without the CUSTOM authentication method, and it works perfectly in any configuration.
Can you guide me a bit more to solve this problem?

**Reply by jeromelaban** [reply]
Make sure to follow the documentation on how to adjust the linker configuration file, [this one specifically](https://platform.uno/docs/articles/features/using-il-linker-webassembly.html).
